Public/Src/Sandbox/MacOs/Sandbox/Src/Kauth/AccessHandler.cpp

```diff
@@ -205,6 +205,22 @@ static bool VNodeMatchesPath(vnode_t vp, vfs_context_t ctx, const char *path)
     return result;
 }
 
+const char* AccessHandler::IgnoreCatalinaDataPartitionPrefix(const char* path)
+{
+    if (!sandbox_->GetConfig().enableCatalinaDataPartitionFiltering)
+    {
+        return path;
+    }
+
+    const char *marker = path;
+    if (strprefix(marker, "/System/Volumes/Data/"))
```

Comment:
Extract `"/System/Volumes/Data/"` into a macro or a const.  Then it will be clearer that here this string has a trailing `/` whereas 2 lines below it doesn't.

Reply:
Refactored into consts.